src/functionalTest/resources/features/F-001 - Get Dictionary/F-001_Test_Base_Data.td.json

```diff
@@ -3,5 +3,7 @@
   "productName": "Translation Service",
   "operationName": "GET dictionary",
   "method": "GET",
-  "uri": "/dictionary"
+  "uri": "/dictionary",
+
+  "s2sClientId": "xui_webapp"
```

Comment:
can you rename and move to match the other endpoints in WLTS-3 commit src/functionalTest/resources/features/common/ts_endpoints/Get_Dictionary_Base.td.json